miasm2/ir/translators/python.py

```diff
@@ -55,6 +56,20 @@ def from_ExprOp(self, expr):
         elif expr.op == "parity":
             return "(%s & 0x1)" % self.from_expr(expr.args[0])
 
+        elif expr.op in ["<<<", ">>>"]:
+            amount_raw = expr.args[1]
+            amount = expr.args[1] % ExprInt(amount_raw.size, expr.size)
+            amount_inv = ExprInt(expr.size, expr.size) - amount
+            if expr.op == "<<<":
+                amount, amount_inv = amount_inv, amount
+            part1 = "(%s >> %s)"% (self.from_expr(expr.args[0]),
+                                   self.from_expr(amount))
+            part2 = "(%s << %s)"% (self.from_expr(expr.args[0]),
+                                         self.from_expr(amount_inv))
+
+            return "((%s & 0x%x) | (%s & 0x%x))" % (part1, int(expr.mask),
```

Comment:
Maybe to be a few less complex and homogeneous with others op translations, one can use the form:
```Python
"((%s | %s) & 0x%x)" % (part1, part2, int(expr.mask))
```